tes_client/communication/tes_connection.py

```diff
@@ -28,9 +28,6 @@
 
```

Comment:
Rename file to `omega_oems` and the class to `OmegaOems`? Can be done in another PR

Reply:
Will do in another PR.